layers/meta-balena-raspberrypi/conf/layer.conf

```diff
@@ -138,6 +145,9 @@ RPI_KERNEL_DEVICETREE:remove:revpi-connect = "bcm2711-rpi-cm4s.dtb"
 RPI_KERNEL_DEVICETREE:remove:revpi-core-3 = "bcm2711-rpi-cm4s.dtb"
 
 PREFERRED_VERSION:linux-raspberrypi = "5.10.%"
+PREFERRED_VERSION_linux-raspberrypi:raspberrypi5 = "6.6%"
```

Comment:
@mtoman I see in the old balena-raspberrypi-temp repo, the machine name used to be `raspberrypi5-64` was the change intentional and will it affect what the slug is in balenaCloud?

Reply:
@shaunmulligan the old temp repo had our clone of meta-raspberrypi in which we added the pi5* machines because the upstream did not have them at that point.
In the meantime the upstream BSP added it and only has the `raspberrypi5` machine which is in 64 bits mode

Reply:
Ah okay, that makes sense. So will we have to migrate these since our current machine slug is different to what is in upstream

Reply:
No, the slug is called `raspberrypi5` so we should be good to go